src/main/kotlin/no/nav/familie/dokument/storage/StonadController.kt

```diff
@@ -41,15 +41,19 @@ class StonadController(@Autowired val storage: MellomLagerService,
     @GetMapping(path = ["/{stonad}"], produces = [MediaType.APPLICATION_JSON_VALUE])
     fun hentMellomlagretSøknad(@PathVariable("stonad") stønad: StønadParameter): ResponseEntity<String> {
         val directory = contextHolder.hentFnr()
-        return ResponseEntity.ok(storage[directory, stønad.stønadKey])
+        return try {
+            ResponseEntity.ok(storage[directory, stønad.stønadKey])
+        } catch (e: GcpDocumentNotFound) {
+            ResponseEntity.notFound().build()
```

Comment:
Snakket med Charlie og vi bestemte for att frontend ønsker 201 i dette tilfelle, hvis mellomlagret ikke finnes. Det er mer som ett søk for å sjekke om det finnes noe, men ikke en